graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/KotlinCodeGenTest.kt

```diff
@@ -1177,35 +1239,8 @@ class KotlinCodeGenTest {
         """.trimIndent()
         val generatedInputString = type.funSpecs.single { it.name == "toString" }.body.toString().trimIndent()
         assertThat(expectedInputString).isEqualTo(generatedInputString)
-    }
-
-    @Test
-    fun generateToStringMethodForDataTypes() {
```

Comment:
Did you mean to remove this since it is redundant with the other tests, or was this a mistake?

Reply:
I meant to remove it. It is not applicable for kotlin, since there is no toString() method generated. The test is actually a java test that is incorrectly placed in kotlin (probably a copy paste error).